lib/src/main/java/space/taran/arklib/data/preview/Preview.kt

```diff
@@ -67,7 +76,8 @@ class PreviewLocator(
     private val previewsDir = root.arkFolder().arkPreviews()
     private val thumbnailsDir = root.arkFolder().arkThumbnails()
 
-    fun fullscreen(): Path = processor.images[id] ?: previewsDir.resolve(id.toString())
+    fun fullscreen(): Path =
+        processor.images[id] ?: previewsDir.resolve(id.toString())
```

Comment:
I know it's just a formatting update here, but since `fullscreen()` method is only being used within `Preview` class at the moment, can we make it `private`?

Reply:
This is a library, this method must be public

Reply:
Yeah, it's used in Navigator most likely